routes/api/courses.js

```diff
@@ -99,6 +99,8 @@ router.put("/:courseId/:studentId", passport.authenticate('jwt', {session: false
         error: "student not found" 
       })
     } else {
+      student.courseIDs.push(courseId);
+      student.save();
```

Comment:
should move this til after you find the course or else you could be adding courses that don't exist.